README.md

```diff
@@ -0,0 +1,148 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[contact\_form\] Contact Form](#-contact_form-contact-form)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [contact_form] <a name="about-project">Contact Form</a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[contact_form]** is a basic project to learn how the code review process will go.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<ul>
+  <li><a href="https://reactjs.org/">HTML</a></li>
+  <li><a href="https://expressjs.com/">CSS</a></li>
+  <li><a href="https://www.postgresql.org/">JS</a></li>
+</ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
```

Comment:
- [ ] Also, I suggest you omit line `43` as it only serves as a guide but should not have been pushed. That way, you keep the README clean, which you'll agree with me is the best 😃

- [ ] Since you don't have live line yet, I suggest you don't make line `45` a hyperlink, so you don't miss lead your viewers. Try something like ` "Under construction or in progress"`. You'd agree that sounds a lot better 🏷️